collectors/ebpf.plugin/ebpf_socket.h

```diff
@@ -94,6 +105,8 @@ typedef enum ebpf_socket_idx {
 // Port range
 #define NETDATA_MINIMUM_PORT_VALUE 1
 #define NETDATA_MAXIMUM_PORT_VALUE 65535
+#define NETDATA_MAXIMUM_CONNECTION_ALLOWED 16384U
+#define NETDATA_MAXIMUM_UDP_CONNECTION_ALLOWED 4096U
```

Comment:
Minor thing, but it would be better to change `CONNECTION` -> `CONNECTIONS`.

Reply:
It was renamed. Thanks!